.vsts-ci.yml

```diff
@@ -142,7 +142,7 @@ stages:
             name: NetCore1ESPool-Svc-Internal
             demands: ImageOverride -equals 1es-ubuntu-2004
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          helixTargetQueue: 'ubuntu.2004.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-amd64-20220502145738-4b2e4c2'
+          helixTargetQueue: ubuntu.2004.amd64.open
```

Comment:
@LakshanF I saw you changed the queue in https://github.com/dotnet/sdk/pull/25060, could you please explain the reasons behind it. 

We need more tooling for the new projects tests, which is only available in this queue. Do you have any objections on changing it back? Thanks